runtime/lua/vim/lsp/extensions.lua

```diff
@@ -0,0 +1,50 @@
+local vim = vim
+local validate = vim.validate
+local log = require 'vim.lsp.log'
+local util = require 'vim.lsp.util'
+local buf = require 'vim.lsp.buf'
+
+local M = {}
+
+-- copied from buf.lua
+local function request(method, params, callback)
+  validate {
+    method = {method, 's'};
+    callback = {callback, 'f', true};
+  }
+  return vim.lsp.buf_request(0, method, params, callback)
+end
+
+-- callback to preview a location in a floating window instead
+-- of jumping to it
+local function preview_location_callback(_, method, result)
+  if result == nil or vim.tbl_isempty(result) then
+    local _ = log.info() and log.info(method, 'No location found')
```

Comment:
you don't actually need the `log.info()` check. Just call `log.info(message)` without storing to a variable.

Reply:
I just copied that from the standard `location` callback.